src/messaging/messaging.ts

```diff
@@ -41,6 +42,12 @@ import MessagingConditionResponse = messaging.MessagingConditionResponse;
 import DataMessagePayload = messaging.DataMessagePayload;
 import NotificationMessagePayload = messaging.NotificationMessagePayload;
 
+// GAPIC Generated client types
+import * as protos from '../generated/messaging/protos/protos';
+import { FcmServiceClient } from '../generated/messaging/src/v1/fcm_service_client'
+import ClientMessage = protos.google.firebase.fcm.v1.IMessage;
```

Comment:
Use the original names `IMessage` and `ISendMessageRequest`. I don't see a reason to change them.

Reply:
My thought was that naming it `ClientMessage` and `ClientSendRequest` would more clearly differentiate between our public types and the generated ones (alternatively something like `GapicMessage` or `GeneratedMessage` etc.). If that isn't necessary I can definitely change to the original names.

Reply:
Let's use the original generated names here. I think the naming convention of `IMessage` and `ISendMessageRequest` itself serves as a good differentiator since we don't use `I` prefixed names in our code.

Reply:
Understood, made the change!